__tests__/helpers/requestMock.ts

```diff
@@ -0,0 +1,30 @@
+import { ClientRequest, request } from "http";
+import IntegrationTestEnvironment from "./integrationTestEnvironment";
+
+export default function requestMock(
+  env: IntegrationTestEnvironment,
+  mockID: string,
+  method: "GET" | "POST" = "GET",
+  body?: any,
+  contentType: string = "application/json"
```

Comment:
`body` is optional but `contentType` not

Reply:
`contentType` has an _initializer_ ("default" value) thus can't be optional (`undefined`)